arch/arm/configs/bcm2709_defconfig

```diff
@@ -648,7 +648,8 @@ CONFIG_SERIAL_AMBA_PL011=y
 CONFIG_SERIAL_AMBA_PL011_CONSOLE=y
 CONFIG_SERIAL_SC16IS7XX=m
 CONFIG_SERIAL_SC16IS7XX_SPI=y
-CONFIG_SERIAL_DEV_BUS=m
+CONFIG_SERIAL_DEV_BUS=y
+CONFIG_SERIAL_DEV_CTRL_TTYPORT=y
```

Comment:
The kernels are going to grow by ~30kB, but in this case I think the eventual end justifies it.